File: app2/src/main/java/ru/neverlands/anclient/postfilter/CastleJs.java
package ru.neverlands.anclient.postfilter;

import ru.neverlands.anclient.utils.Russian;

public class CastleJs {
    public static byte[] process(byte[] array) {
        String js = Russian.getString(array);
        js = js.replace("document.location = goloc;", "window.open(goloc, 'main');");
        return Russian.getBytes(js);
    }
}

File: ANClient/PostFilter/CastleJs.cs
using ANClient.Helpers;
using ANClient.Properties;

namespace ANClient.PostFilter
{
    internal static partial class Filter
    {
        private static byte[] CastleJs(byte[] array)
        {
            var html = Russian.Codepage.GetString(array);
            html = Resources.json2 + " " + html;
            return Russian.Codepage.GetBytes(html);
        }
    }
}
